src/assets/js/components/options/sign-in.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { useState } from 'react';
+
+import { callApi } from '@/main.mjs';
+import { useDispatch } from 'react-redux';
+import { signIn } from '@/features/auth/authSlice.js';
+
+import { TokenHelpText } from '@/components/options/token-help-text.jsx';
+
+const SignInError = () => {
+  return (
+    <div className='alert alert-danger mt-2' role='alert'>
+      Unable to sign in. Check your credentials and internet connectivity,
+      then try again.
+    </div>
+  );
+}
+
+export const SignIn = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [showSignInError, setShowSignInError] = useState(false);
+  const [token, setToken] = useState('');
+  const dispatch = useDispatch();
+
+  const getSignUpLink = () => {
+    const baseUrl = 'https://login.screenlyapp.com/sign-up';
+    const queryParams = `next=${window.location.href}`;
+    return `${baseUrl}?${queryParams}`;
+  };
+
+  const handleSignIn = async (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+
+    try {
+      await callApi(
+        'GET',
+        'https://api.screenlyapp.com/api/v4/assets/',
+        null,
+        token
+      );
+
+      await browser.storage.sync.set({ token: token });
+
+      setShowSignInError(false);
+    } catch (error) {
+      setShowSignInError(true);
+      return;
+    } finally {
+      setIsLoading(false);
+    }
+
+    setIsLoading(false);
+    dispatch(signIn());
+  }
+
+  return (
+    <>
+      <div>
```

Comment:
The return is already wrapped inside a <div>.

You can either use <></> or a parent div to return the JSX